### DISCUSSION TITLE
[M-Z] (2024/25) Foglio 6 - Esercizio 15

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/algebra#577] [Esercizi M-Z]
![es15](https://github.com/user-attachments/assets/35e8fe95-ddc2-4940-895a-f64f9009a201)

**Comment by alem1105** [comment]
in quella sotto ci andava il -1, ho solo riscritto l'inverso

![image](https://github.com/user-attachments/assets/9051eebd-4e81-414a-a002-0189f4769938)
